### PR TITLE
feature login

### PR DESCRIPTION
**Pull request by c3rberuss**

### CODE REVIEW

File: app/src/main/java/com/developers_recipes/recetario/models/AuthModel.kt [hunk 1]
```diff
@@ -0,0 +1,3 @@
+package com.developers_recipes.recetario.models
+
+class AuthModel(var email: String, var passwword: String)
```
Comment (alelopez07) [inline app/src/main/java/com/developers_recipes/recetario/models/AuthModel.kt]: data class?

File: app/src/main/res/layout/activity_login.xml [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+            android:layout_height="match_parent"
+            android:orientation="vertical"
+            android:padding="16dp">
+
+            <TextView
+                android:id="@+id/title"
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                android:layout_marginBottom="16dp"
+                android:text="Login"
```
Comment (alelopez07) [inline app/src/main/res/layout/activity_login.xml]: Don't write your strings here. You need to use strings.xml and write your id_string here. @string/text_something.

File: app/src/main/java/com/developers_recipes/recetario/utils/Constants.kt [hunk 3]
```diff
@@ -1,10 +1,13 @@
 package com.developers_recipes.recetario.utils
```
Comment (alelopez07) [inline app/src/main/java/com/developers_recipes/recetario/utils/Constants.kt:2]: Would be a constants.kt file and not a class? The "const" keyword, has a static definition by default.

**Review by alelopez07 — COMMENTED** [review]
Please, check my comments in your code lines.

File: app/src/main/java/com/developers_recipes/recetario/models/AuthModel.kt (hunk 1 above)
Comment (c3rberuss) [reply app/src/main/java/com/developers_recipes/recetario/models/AuthModel.kt]: its solved

File: app/src/main/res/layout/activity_login.xml (hunk 2 above)
Comment (c3rberuss) [reply app/src/main/res/layout/activity_login.xml]: its solved

File: app/src/main/java/com/developers_recipes/recetario/utils/Constants.kt (hunk 3 above)
Comment (c3rberuss) [reply app/src/main/java/com/developers_recipes/recetario/utils/Constants.kt:2]: its solved

**Pull request closed without merging**